.github/workflows/i18n-nightly-push.yml

```diff
@@ -18,6 +22,17 @@ jobs:
           node-version: '14.x'
           registry-url: 'https://registry.npmjs.org'
 
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Install Task
+        uses: arduino/setup-task@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          version: 3.x
+
```

Comment:
`yarn run i18n:push`:

Installing the entire IDE2 dependencies to push the translations makes no sense. Hence adding `Golang` and `Task` is also unnecessary.

Reply:
This is the same for the nightly and weekly translation pushes and for the theme.

Instead of adding unnecessary workflow dependencies, we should clean up these tasks and make them IDE2/Theia independent.

Reply:
I agree that the should be no reason to pull in the Arduino CLI dependency. I didn't set up this infrastructure and I don't know anything about it other than that it is failing right now.

My proposal gets them back to a working state, even if just as inefficient as before.

If someone wants to clean the system up, great, but I don't have time to figure that all out.